src/MarkdownRenderingPlugin.ts

```diff
@@ -18,19 +20,20 @@ export function PageNameGenerator(label: string) {
   label = NoteWorkspace.stripExtension(label);
 
   // Either use the first result of the cache, or in the case that it's empty use the label to create a path
-  let path: string =
-    results.length != 0 ? results[0].path : NoteWorkspace.noteFileNameFromTitle(label);
+  let path = results.length != 0 ?
+    workspace.asRelativePath(results[0].path, false) :
+    NoteWorkspace.noteFileNameFromTitle(label);
 
   return path;
 }
 
 // Transformation that only gets applied to the page name (ex: the "test-file.md" part of [[test-file.md | Description goes here]]).
-export function postProcessPageName(pageName: string) {
+function postProcessPageName(pageName: string) {
```

Comment:
Are these function name changes just cleanup to match the names in `pluginSettings`?